style.css

```diff
@@ -0,0 +1,268 @@
+* {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+}
+body {
+    margin: 0 auto;
+    height:100%;
+}
+nav {
+    margin: 0;
+    padding: 0;
+    background-color: #111111;
+    width: 100%;
+    position: sticky;
+    top: 0;
+    color: #dddddd;
+    font-family: 'Lato', sans-serif;
+    z-index:5;
+}
+#navwrapper {
+    display:-webkit-box;
+    display:-ms-flexbox;
+    display:flex;
+    width: 65%;
```

Comment:
if you make this a maximum width, the smaller sizes scale much better and dont need media queries

Reply:
resolved in 3df4c38